### PR TITLE
New face

### PR DESCRIPTION
**Pull request by just-boris**
related #721

### CODE REVIEW

**Comment by just-boris** [comment]
retest this please

File: allure-report-face/src/index.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import './plugins/graph';
+import './plugins/timeline';
+
+import DescriptionView from './plugins/testcase-description/DescriptionView';
+allurePlugins.addTestcaseBlock(DescriptionView, {position: 'before'});
+
+App.start();
+Backbone.history.start();
+
+window.jQuery = $;
```
Comment (lanwen) [inline allure-report-face/src/index.js]: why u need jQuery in global scope?

Comment (just-boris) [inline allure-report-face/src/index.js]: It was needed me for debugging.

Anyway, till release I am going to expose `Backbone`, `Underscore` and `Handlebars` as globals too. It the only way how to reuse them in plugins.

Comment (just-boris) [inline allure-report-face/src/index.js]: Also, there is a Chrome Extension - [Marionette-inspector](https://github.com/marionettejs/marionette.inspector) which also needs globally available that libraries

**Comment by lanwen** [comment]
Code looks great now. This pr increases maintainability over 9000

**Comment by baev** [comment]
I guess we should show only the icon for language switch in case menu is collapsed.

**Comment by baev** [comment]
`LinksView.js` why the class name is `IssuesView`?

**Comment by baev** [comment]
see #776

**Pull request closed without merging**